Automattic/jetpack
# Instant Search: search from any page on the site

Allow the end user to do instant search and filtering no matter where they are on the site. Filters are always displayed and the end user can always combine filters and search queries.

#### Testing instructions:
* Add define( "JETPACK_SEARCH_PROTOTYPE", true ); to your wp-config.php.
* Ensure that your site has the Jetpack Pro plan and Jetpack Search enabled.
* Add a Jetpack Search widget to the Search page sidebar and configure some filters.
* Here are the cases I've been testing:

```
 - on search page e.g. http://gibrown.wpsandbox.me/main/?blog_id=20115252&orderby=relevance&order=DESC&s=related%20psts
	 - filter
	 - filter then cancel search
	 - filter then search
	 - filter then filter
 - on home page e.g. http://gibrown.wpsandbox.me/main/?blog_id=20115252
	 - filter
	 - search
	 - filter then unfilter
	 - search then esc
	 - filter then search
	 - filter then filter
	 - search then filter
 - on a post page e.g.  http://gibrown.wpsandbox.me/main/?blog_id=20115252&p=1241
	 - filter
	 - filter then unfilter
	 - search
	 - search then esc
	 - filter then search
	 - filter then filter
	 - search then filter

```


#### Proposed changelog entry for your changes:

None.
